changelogs/unreleased/1142-daved

```diff
@@ -0,0 +1 @@
+Clarify restore log when object unchanged
```

Comment:
can you rename this file to `1153-daved`? we're currently using the PR # rather than the issue #.

Reply:
No problem. Is it safe to assume that I should re-squash after?

Reply:
yes please!

Reply:
Yes

Reply:
@daved would you be able to get to this in the next day or two? thanks!

Reply:
@skriss Sure! Sorry for the delay, I had taken the weekend+Friday off from code. If I screwed anything up I'll be keeping an eye on this throughout the day.

Reply:
awesome, thanks @daved!